Core/Util/GameManager.cpp

```diff
@@ -223,7 +224,7 @@ bool GameManager::InstallGame(std::string zipfile, bool deleteAfter) {
 		const char *fn = zip_get_name(z, i, 0);
 		// Note that we do NOT write files that are not in a directory, to avoid random
 		// README files etc.
-		if (strchr(fn, '/') != 0) {
+		if (strchr(fn, '/') != 0 || strchr(fn, '/') == 0) {
```

Comment:
This line pretty much becomes  ```if (true) ``` and is just useless. This should make you think about why it's there in the first place. For example, the above comment becomes invalid in the case of a ISO or CSO.

I think the logic overall needs cleaning up.

Reply:
Ok, I'll work on the implementation I talked about in a previous comment then and recommit. What I'm thinking is, leaving the eboot behavior as was originally designed, and only do something else if its a a iso/cso. if you prefer this course of action, please let me know, Thanks.

Reply:
I cleaned the logic, while leaving the original intent for pbp files